gc/verbose/handler_standard/VerboseHandlerOutputStandard.cpp

```diff
@@ -243,6 +244,44 @@ MM_VerboseHandlerOutputStandard::getCycleType(uintptr_t type)
 	return cycleType;
 }
 
+const char *
+MM_VerboseHandlerOutputStandard::getConcurrentTypeString(uintptr_t type)
+{
+	const char* cycleType = NULL;
+	switch (type) {
+	case OMR_GC_CYCLE_TYPE_GLOBAL:
+		cycleType = "global mark";
+		break;
+	case OMR_GC_CYCLE_TYPE_SCAVENGE:
+		cycleType = "scavenge";
+		break;
+	default:
+		cycleType = "unknown";
+		break;
+	}
+
+	return cycleType;
+}
+
+const char *
+MM_VerboseHandlerOutputStandard::getCardCleaningReasonString(uintptr_t type)
```

Comment:
Please move this & `handleConcurrentMarkEnd` to be under `#if defined(OMR_GC_MODRON_CONCURRENT_MARK)`